projects/ontimize-web-ngx/src/lib/components/image/o-image.component.ts

```diff
@@ -144,6 +129,8 @@ export class OImageComponent extends OFormDataComponent implements OnInit, OnDes
     } else {
       this.value = new OFormValue(undefined);
     }
+
+    this.src = this.getSrcValue();
```

Comment:
Pay attention, if val is instanceof OFormValue there is a return on line 118 and the value of src is not stored!